example/demo/src/ui-controls/ui.tsx

```diff
@@ -50,11 +52,51 @@ function UI() {
         <Button onPress={navigate('list')}>List</Button>
         <Button onPress={navigate('toast')}>Toast</Button>
         <Button onPress={navigate('haptics')}>Haptics</Button>
+        <Button onPress={navigate('svg')}>SVG</Button>
       </YStack>
     </>
   )
 }
 
+function SVGExample() {
+  return (
+    <YStack gap="$8" padding="$4">
+      <YStack>
+        <H4>SVG</H4>
+        <Svg height="100" width="100" viewBox="0 0 100 100">
+          <Circle cx="50" cy="50" r="45" stroke="blue" strokeWidth="2.5" fill="green" />
+        </Svg>
+      </YStack>
+      <YStack>
+        <H4>SVG from URI</H4>
+        <SvgCssUri
+          width="100"
+          height="100"
+          uri="https://dev.w3.org/SVG/tools/svgweb/samples/svg-files/ruby.svg"
+          onError={toast('Failed to load SVG from URI')}
+        />
+      </YStack>
+      <YStack>
+        <H4>SVG string</H4>
+        <SvgCss
```

Comment:
calling this `SvgCss` feels off to me. the SVG here is XML so maybe it should be called SvgXml

Same with `SvgCssUri`. maybe should just be `SvgUri` (because people using a svg from a URL won't actually care that it is a xml)

Reply:
Yeah I haven't looked into it too much, just re-exported everything from SVG without doing anything extra. I find some of these makes quite verbose too, but that's the API. I would be skeptical about changing it too much to avoid confusion. But maybe we can build a higher order SVG component that does that?

Reply:
I guess here `SvgCss` is b/c there's `CSS` in that SVG string and they use different components for it, likely to optimise.

Reply:
It's `SvgXml` when there's no `CSS` inside. Should've been `SvgCssXml` to keep it consistent I guess but

Reply:
> Same with SvgCssUri. maybe should just be SvgUri (because people using a svg from a URL won't actually care that it is a xml)

All of these are available, depending on whether XML contains <style />

From uri:
- SvgUri  <- no CSS
- SvgCssUri  <- with CSS

From string:
- SvgXml <- no CSS
- SvgCss <- with CSS